go/client/cmd_account_recover_username.go

```diff
@@ -0,0 +1,73 @@
+// Copyright 2017 Keybase, Inc. All rights reserved. Use of
+// this source code is governed by the included BSD license.
+
+package client
+
+import (
+	"fmt"
+
+	"golang.org/x/net/context"
+
+	"github.com/keybase/cli"
+	"github.com/keybase/client/go/libcmdline"
+	"github.com/keybase/client/go/libkb"
+	keybase1 "github.com/keybase/client/go/protocol/keybase1"
+)
+
+type CmdAccountRecoverUsername struct {
+	libkb.Contextified
+	email string
+	phone keybase1.PhoneNumber
+}
+
+func NewCmdAccountRecoverUsername(cl *libcmdline.CommandLine, g *libkb.GlobalContext) cli.Command {
+	return cli.Command{
+		Name:  "recover-username",
+		Usage: "Recover username via email or sms.",
```

Comment:
for the few occurrences of "SMS" I found in Keybase, it's always been uppercased